controllers/loadtest_controller_test.go

```diff
@@ -759,3 +762,87 @@ var _ = Describe("Pod Creation", func() {
 		})
 	})
 })
+
+var _ = Describe("getRequeueTime", func() {
+	var test *grpcv1.LoadTest
+	var reconciler LoadTestReconciler
```

Comment:
A reconciler is a large object, can we make this a pointer?

```suggestion
	var reconciler *LoadTestReconciler
```